pkg/virt-controller/services/template_test.go

```diff
@@ -53,16 +53,20 @@ import (
 	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
 )
 
-// TODO: The memory module is different in different cpu arch, apply multi-arch testing for memory contraint
 var _ = Describe("Template", func() {
+	var configFactory func(string) (*virtconfig.ClusterConfig, cache.SharedIndexInformer, TemplateService)
 	var qemuGid int64 = 107
+	var hostArch = runtime.GOARCH
```

Comment:
I personally think that the unit tests should always lead to the same results, independent of the execution platform. Should this be hardcoded to amd64?

one alternative for all the tests which should lead to the same result, independent of the architecture, would be generating `It` specs for each architecture like explained here: https://onsi.github.io/ginkgo/#patterns-for-dynamically-generating-tests

Reply:
Ok, for now, I just hardcode it to amd64. And I have read through the dynamically generating tests, I think it may help if we expand more arch-specific tests.